specs/psl-sql-conversion.md

```diff
@@ -0,0 +1,459 @@
+# Implementation Spec: Mapping a Prisma Schema to a SQL Database
+
+The schema of the underlying database can be deducted from a Prisma schema. This document describes how this is done for SQL databases.
+
+
+## Mapping Models
+
+### Mapping a model with a single Id field
+Prisma Schema Example with CUID:
+```groovy
+model Blog {
+    id        String @id @default(cuid())
```

Comment:
I think we should mention that the functions for `cuid` and `uuid` have no representation in the database schema.